runtime/near-vm-logic/src/logic.rs

```diff
@@ -1263,30 +1263,29 @@ impl<'a> VMLogic<'a> {
             }
         };
 
-        let mut pubkeys: Vec<blst::min_pk::PublicKey> = vec![];
+        let mut pubkeys: Vec<blst::min_pk::PublicKey> =
+            Vec::with_capacity(pubkeys_cnt.try_into().unwrap());
```

Comment:
Please also rename `pubkeys_cnt`->`num_pubkeys` to comply with the naming conventions used.

Reply:
Fixed 56ac88a1c983f7c600c0e8990396053f99e0cd81